opm/grid/common/GeometryHelpers.hpp

```diff
@@ -148,10 +148,13 @@ namespace Dune
             for (int i = 0; i < num_points; ++i) {
                 Point tet[4] = { cell_centroid, face_centroid, points[i], points[(i+1)%num_points] };
                 double small_volume = std::fabs(simplex_volume(tet));
-                assert(small_volume > 0);
+                if (small_volume == 0.) {
+                    small_volume = 0.;
+                }
```

Comment:
This seems to have no effect.

Reply:
Sorry, that is a debug code. I will remove it.